src/components/MovieDetails/MovieDetails.jsx

```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React from 'react'
+import './MovieDetails.css'
 
-const MovieDetails = () => {
-  return <h1>Placeholder for movie details</h1>;
-};
+const MovieDetails = ({ currentMovie, closeModal }) => {
+  return (
+    <div className="movieDetailsModal" style={{backgroundImage: `url(${currentMovie['backdrop_path']})`, backgroundPosition: 'center', backgroundSize: 'cover'}}>
```

Comment:
I see you've got some inline styling here - but you also have separate CSS files too. I'd encourage you to pick one and stick with it so that your fellow devs always know where to go when adjusting styles

Reply:
I wasn't able to figure out how to make a `background img` dynamic without having it in line so I could interpolate the url. Is there a way to do that directly from the stylesheet or should I leave that here and move `backgroundPosition` and `backgroundSize` only?